File: ws/villagomez/u2/ws21zooWithInheritance/ws21Zoo/src/main/java/ec/espe/ws21zoo/model/Cage.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.espe.ws21zoo.model;

/**
 *
 * @author Domenica Villagomez, CommitCrew, DCCO-ESPE
 */
public class Cage {
    private int id;
    private String type;

    @Override
    public String toString() {
        return "Cage{" + "id=" + id + ", type=" + type + '}';
    }

    public Cage(int id, String type) {
        this.id = id;
        this.type = type;
    }

    /**
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * @return the type
     */
    public String getType() {
        return type;
    }

    /**
     * @param type the type to set
     */
    public void setType(String type) {
        this.type = type;
    }
    
    
}

File: hw/ccedeno/u2/HW18ZooInPythonJavaScript/ZooInJavaScript/ec.edu.espe.Zoo.model/Cage.js
class Cage {
    constructor(id, type) {
      this.id = id;
      this.type = type;
    }
  
    toString() {
      return `Cage{id=${this.id}, type=${this.type}}`;
    }
  
    // Getters and Setters
    getId() {
      return this.id;
    }
  
    setId(id) {
      this.id = id;
    }
  
    getType() {
      return this.type;
    }
  
    setType(type) {
      this.type = type;
    }
  }
  
  module.exports = Cage;
  